src/pages/Check/index.jsx

```diff
@@ -13,12 +13,31 @@ const Check = () => {
   const history = useHistory()
   const { packages, loading, error } = useGetPackagesIdByCode(packageCode?.packageCode)
 
-  const toDelivery = async e => {
+  // eslint-disable-next-line complexity
+  const toDelivery = async (/** @type {{ packageCode: String; }} */ e) => {
     if (e?.packageCode && !error && !loading) {
-      const foundPackage = (await packages.length) > 0
+      const foundPackage = packages.length > 0
+
       if (foundPackage) {
-        const { id } = await packages[0]
-        history.push(`/delivery/${id}`)
+        const { id, order_status } = await packages[0]
+        // eslint-disable-next-line camelcase
+        switch (order_status) {
+          case 'ready':
+            history.push(`/delivery/${id}`)
+            break
+          case 'collected':
+            history.push(`/delivery/${id}`)
+            break
+          case 'in_travel':
+            history.push(`/delivery/${id}`)
+            break
+          case 'destination_reached':
+            history.push(`/confirm/${id}`)
+            break
+          default:
+            alert('el paquete ya fue entregado')
+            break
```

Comment:
What status have the package in this line?

Reply:
here the status is rated or package confirmed

Reply:
Please check: https://github.com/Azordev/dasher-user/pull/164#discussion_r754200970 .

On the other hand, wouldn't it be better to specify so that the states of the package are documented?

Or if new states are added to the package it is easier to change it.

instead 
```
default:
    alert('el paquete ya fue entregado'); //PD: Please add uppercase at the start 'El paquete*'
```

use

```
case 'is_rated':
case 'package_confirmed':
   alert('El paquete ya fue entregado')
   break;
```

Then when other developer want add other new status, he not need search what status are in 'default', instead he only need add the need status